go-controller/pkg/util/node_annotations.go

```diff
@@ -97,6 +97,9 @@ const (
 	// OVNNodeHostCIDRs is used to track the different host IP addresses and subnet masks on the node
 	OVNNodeHostCIDRs = "k8s.ovn.org/host-cidrs"
 
+	// OVNNodePrimaryDPUAddr is used to track the primary DPU address on the node
+	OVNNodePrimaryDPUAddr = "k8s.ovn.org/primary-dpu-addr"
```

Comment:
would this actually make more sense as primary-dpu-host-addr?

Reply:
updated + helpers